develop/tutorials/articles/260-themes-and-layout-templates/07-using-developer-mode-with-themes.markdown

```diff
@@ -52,41 +68,20 @@ file has the property setting:
 `include-and-override=portal-developer.properties`, which enables Developer
 Mode. 
 
-Liferay IDE sets most of the changes needed for Developer mode
-
-
-FreeMarker Templates for themes and web content are cached by default. Therefore, 
-any changes you make to your FreeMarker theme templates won't show up 
-immediately. You can change this behavior through System Settings. Follow these 
-steps to update the system setting:
-
-1.  Open the Control Panel and goto *Configuration* &rarr; *System Settings*.
-
-2.  Click the *Foundation* tab and select *FreeMarker Engine*.
-
-3.  By default, the *Resource modification check*, the time in milliseconds that 
-    the template is cached, is set to 60. Set this value to 0 to disable caching. 
--->
-Learn how to manually set Developer Mode next.
+Most of the configuration is provided by the `portal-developer.properties` file, 
+but you still have to configure the FreeMarker template setting. Follow the 
+steps in the [Configuring FreeMarker System Settings](/develop/tutorials/-/knowledge_base/7-0/using-developer-mode-with-themes#configuring-freemarker-system-settings) 
+section to configure the FreeMarker template cache, or move to the next section 
+to learn how to manually set Developer Mode in your app server. 
 
 ## Setting Developer Mode for Your Server Using portal-developer.properties [](id=setting-developer-mode-for-your-server-using-portal-developer-properties)
 
-To set Developer Mode manually, you must add the `portal-developer.properties`
+To set Developer Mode manually, You must add the `portal-developer.properties` 
 file to your application server's launch configuration. Since each application 
 server has a different configuration file or UI to specify system properties, 
 you must follow your application server's specific method for adding the 
 `portal-developer.properties` file to the server's launch configuration. 
```

Comment:
I don't know that. It's my assumption. I assume Portal is explicitly looking for `portal-ext.properties` and it doesn't depend on the app server.